BLSQ/iaso
# IA-1483 query builder

No problem here, only a new feature: allow to search in submission fields using json logic

## Self proof reading checklist


## Changes

- using brand new components from bluesquare-components [PR](https://github.com/BLSQ/bluesquare-components/pull/80)
- use possible fields and form descriptor to get fields label and value for select and multiselect
- map possible fields with query builder fields



## How to test

Open submissions page and select a unique form.
A new field appear beneath.
Click on it and ply with the query builder to search into fields value
Check that the results are corrects

## Print screen / video

![Screenshot 2022-10-04 at 15 19 39](https://user-images.githubusercontent.com/12494624/193836208-844c57df-56c3-499f-9dd2-887e28b6b906.png)
![Screenshot 2022-10-04 at 15 19 30](https://user-images.githubusercontent.com/12494624/193836216-70586a7b-63c8-4dc8-b556-b90bb0defcd7.png)
![Screenshot 2022-10-04 at 15 19 21](https://user-images.githubusercontent.com/12494624/193836224-82f5c5ea-2b5e-44ca-8d5d-68096f8ab502.png)
![Screenshot 2022-10-04 at 15 19 09](https://user-images.githubusercontent.com/12494624/193836226-0ddb2cbf-8952-4a91-8126-e1a102f706fc.png)


## Notes

I'm using by default last_version on the form selected.
Other operators such as proximity, contains, ... are disabled for now because backend is not supporting it
